### PR TITLE
Executors, Callable and Future

### PR DESCRIPTION
**Pull request by Yanetta**

### CODE REVIEW

**Review by DadizhaO — APPROVED** [review]
I don't see any mistakes. Good work!

File: src/main/java/executorsCallableFuture/CallableClass.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    private Resource res;
+    private String name;
+
+    public CallableClass (Resource res, String name) {
+        this.res = res;
+        this.name = name;
+    }
+
+    @Override
+    public Resource call() throws Exception {
```
Comment (SerhiiPasko) [inline src/main/java/executorsCallableFuture/CallableClass.java]: Why you throws top-level Exception???

File: src/main/java/executorsCallableFuture/Main.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
+package executorsCallableFuture;
+
+import java.util.concurrent.*;
+
+public class Main {
+    public static void main(String[] args) {
+        Resource res = new Resource(0);
+        ExecutorService ex = Executors.newFixedThreadPool(3);
+
+        FutureTask<Resource> f1 =new FutureTask<Resource>(new CallableClass(res, "name1"));
```
Comment (SerhiiPasko) [inline src/main/java/executorsCallableFuture/Main.java:10]: forEach

**Review by SerhiiPasko — APPROVED** [review]
Read comments

**Pull request merged**